src/components/cycle-section/cycle-section/LearningUnitsCard.js

```diff
@@ -0,0 +1,16 @@
+import { Card } from 'primereact/card';
+import styles from './CycleSection.module.scss';
+
+const LearningUnitsCard = ({ learningUnits }) => {
+  return (
+    <Card className={styles.learningUnitsCard} title="Learning units">
+      <ul>
+        {learningUnits.map((lu) => (
+          <li key={lu.id}>{lu.name}</li>
```

Comment:
This li tag could be a Link to the learning units.

Reply:
Done!